app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/welcome_screen_wallpaper"
+    tools:context=".appMode.ModeChoiceActivity">
+
+
+    <Button
+        android:id="@+id/manager_button"
+        android:layout_width="146dp"
+        android:layout_height="47dp"
+        android:layout_marginStart="10dp"
+        android:layout_marginTop="110dp"
+        android:layout_marginEnd="30dp"
+        android:text="@string/manager_option"
+        android:textColor="#000000"
+        android:onClick="onAppModeChosen"
```

Comment:
this is interesting. i usually try to avoid mixing XML and code like that, but it isnt inherently wrong. consider using `setOnClickListener` on `onCreate`, but take this just as a proposal. comment your choice

Reply:
Think I'll leave it this way